lib/rbac/roles.rb

```diff
@@ -39,6 +39,15 @@ def delete(role)
       end
     end
 
+    def self.assigned_role?(role_name)
+      opts = { :limit => 100,
+               :scope => 'principal' }
+
+      RBAC::Service.call(RBACApiClient::RoleApi) do |api_instance|
+        RBAC::Service.paginate(api_instance, :list_roles, opts).any? { |role| role.name == role_name }
```

Comment:
@lindgrenj6 I am thinking we could just change the above line to
```
RBAC::Service.paginate(api_instance, :list_roles, opts).count > 0
```

Reply:
Changed to `.count.positive?` due to rubocop complaining